README.md

```diff
@@ -26,8 +26,18 @@ end
 # app/jobs/compliance_job.rb
 class ComplianceJob
   def perform(record, value)
-    production_env? &&
-      EasyCompliance::Client.submit(record: record, value: value)
+    return unless production_env?
+
+    check = EasyCompliance::Client.submit(record: record, value: value)
+    check.hit?   # true/false
+
+    check.status # http response status e.g. 200
+    check.body   # http response body
```

Comment:
I suspect this is just an example method that describes how to use the response from easy compliance. If so that's fine for me. Just asking because there are no code changes...

Reply:
yes, I wanted to know what one actually does with the result. We have some internal story where we might need to touch things here.